plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/helpers.ts

```diff
@@ -29,48 +23,3 @@ export const generateBackstageUrl = (config: Config, path: string): string => {
   }
   return externalUrl + path;
 };
-
-export const http = async (
```

Comment:
I moved this into a class, so that I could pass it a reference to the authorizer.